src/Components/FeelingRoom/angryAnimations.js

```diff
@@ -0,0 +1,67 @@
+import gsap from 'gsap'
+
+export const angryFeelingAnimation = () => {
+  let angryTimeline = gsap.timeline({repeat: 10});
+  for (var i = 0; i <= 9; i++) {
+    angryTimeline.from(`.square${i}`, {
+      duration: .5,
+      y: "-50vh",
+      display: "flex",
+    })
+    .to(`.square${i}`, {
+      duration: 1.75,
+      y: "60vh",
+      ease: "bounce",
+      backgroundColor: 'hsl(329, 57%, 25%)',
+    })
+    .add("shake")
+    .to(`.square${i}`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part1`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part2`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part3`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .to(`.square${i} #part4`, {
+      backgroundColor: 'hsla(329, 57%, 25%)',
+    }, "shake")
+    .add("explode")
```

Comment:
Ah, I see, this is how you layer on the animations and such in the timeline! This looks really, really good.